itez/beneficiary/models.py

```diff
@@ -1,6 +1,9 @@
 from django.contrib.gis.db import models
 from django.db.models.fields.related import create_many_to_many_intermediary_model
 from django.utils.translation import gettext_lazy as _
+from imagekit.processors import ResizeToFill
+from imagekit.models import ProcessedImageField
```

Comment:
a single line seperating django imports from custom local imports

Reply:
Done, I did the same for other files as well.